Cake.Recipe/Content/parameters.cake

```diff
@@ -373,6 +373,8 @@ public static class BuildParameters
         string repositoryName = null,
         string appVeyorAccountName = null,
         string appVeyorProjectSlug = null,
+        string masterBranch = "master",
```

Comment:
Just a reflection, Shouldn't new parameters come last? just in case someone isn't using names parameters.

Reply:
@devlead yeah, that is probably a good idea.  @RLittlesII are you able to make this change?  At some point, all of this will change to use TypedContexts, but until then, I think we should follow this suggestion.

Reply:
@devlead Your absolutely right.  I forgot that just because they are defaults, doesn't mean people might not be using named params. Fixed.